web_tool_script_1.R

```diff
@@ -5,19 +5,7 @@ options(encoding = "UTF-8")
 # devtools::install_github("2DegreesInvesting/r2dii")
 # install.packages("fst")
```

Comment:
should these commented out install lines also be removed from here and/or added to `dry/packages.R`?

Reply:
I'll submit a separate PR to remove dead code like this one.

Reply:
https://github.com/2DegreesInvesting/PACTA_analysis/issues/144